### ISSUE TITLE
Manually defining a port for TV

### ISSUE THREAD

**Issue by khurrraam** [tavicu/homebridge-samsung-tizen#220]
Hello!

I've a small but a bit peculiar request. I'm running HomeBridge in Docker behind Traefik reverse proxy. Now, this means that I have to define the ports in the docker-compose file.

From the logs:
[5/12/2020, 1:55:23 AM] Homebridge is running on port 51652.
[5/12/2020, 1:55:23 AM] Bedroom TV is running on port 41217.
[5/12/2020, 1:55:23 AM] Please add [Bedroom TV] manually in Home app. Setup Code: 103-85-018

In the Config:
Now, the thing is, the TV port keeps changing at every restart. I thought the following might work for example as:

        {
            "name": "SamsungTizen",
            "devices": [
                {
                    "name": "Bedroom TV",
                    "ip": "192.168.31.199",
                    "port": 41217,
                    "mac": "XX:XX:XX:XX:XX:XX"
                }
            ],
            "platform": "SamsungTizen"
        }

So, in Traefik if I define that port which needs to me to restart the container, then the port randomises to something else. Would it be possible to be able to define port manually?

**Comment by khurrraam** [comment]
Here is how we solve announcing Homebridge avahi services while having Traefik.

https://www.devwithimagination.com/2020/04/11/running-homebridge-on-docker-without-host-network-mode-additional-accessories/

Any input would be much appreciated.

**Comment by khurrraam** [comment]
Okay, I'm starting to see that this might not be an issue as such related to your work. My apologies about the request. I guess, this whole Traefik reverse proxy stuff isn't ideal for HomeBridge given that accessories are added and they get announced on a range of random ports.

I tried out and I noticed that HOOBS (a derivative layer built on top of HomeBridge), lets you define the port range and then, does give the Samsung TV (given it is the first and only accessory) a fixed port (first port from the range, i.e. 35000-55000).

I guess, we need to know the WOL for the TV and then, fine tune Traefik or just should give up on Traefik + Homebridge combination.
